### PR TITLE
Make requested changes to course content CSS

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd @mondiaz

Default CSS changes for Stanford courses requested by Instructional Designers.

Before:
screen shot 2017-12-20 at 12 27 41 pm

screen shot 2017-12-20 at 12 27 49 pm

After:
screen shot 2017-12-15 at 3 29 43 pm

screen shot 2017-12-15 at 3 30 01 pm

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Can you attach before-shots of this page too?

File: lagunita/lms/static/sass/_overrides.scss [hunk 1]
```diff
@@ -215,6 +215,130 @@ body.discussion {
...
+        line-height: 1.38em;
+        margin-top: 1em;
+        margin-bottom: .3em;
+    }
+
+    .blue {
+        color: #476bb2;
+    }
+
+    /* Bullets for unordered lists <ul> */
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/_overrides.scss]: Superfluous comments are superfluous ;)

File: lagunita/lms/static/sass/_overrides.scss [hunk 2]
```diff
@@ -215,6 +215,130 @@ body.discussion {
...
+        list-style-type: disc;
+    }
+    ul ul ul ul ul li {
+        list-style-type: circle;
+    }
+    ul ul ul ul ul ul li {
+        list-style-type: square;
+    }
+
+    /* Bullets for ordered lists <ol> */
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/_overrides.scss]: Superfluous comments are superfluous ;)

**Comment by caesar2164** [comment]
Done @stvstnfrd.

File: lagunita/lms/static/sass/_overrides.scss [hunk 3]
```diff
@@ -215,6 +215,128 @@ body.discussion {
     }
 }
 
+#seq_content {
+    font-size: 16px !important;
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/_overrides.scss]: Let's use more specific selectors and remove the `!important`s.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]
Only technical note is to remove the `!important`s.

Has this been run by Chris for UX approval?

**Comment by caesar2164** [comment]
Ok, @stvstnfrd removed !importants, and pinged Chris.

File: lagunita/lms/static/sass/_overrides.scss [hunk 4]
```diff
@@ -216,8 +216,8 @@ body.discussion {
 }
 
 #seq_content {
-    font-size: 16px !important;
-    letter-spacing: normal !important;
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/_overrides.scss]: If this "just works", why did you add the `!important` in the first place?

Comment (caesar2164) [reply lagunita/lms/static/sass/_overrides.scss]: it was insurance...

Comment (stvstnfrd) [reply lagunita/lms/static/sass/_overrides.scss]: Okay, let's try to avoid "just in cases" next time ;)

Anyway, +1

**Review by stvstnfrd — APPROVED** [review]
